pkg/operator/ceph/file/mds/spec.go

```diff
@@ -161,3 +156,27 @@ func deleteMdsDeployment(context *clusterd.Context, namespace string, deployment
 	}
 	return nil
 }
+
+func (c *Cluster) addMDSStandbyFlags() []string {
+	var args []string
+
+	if !c.clusterInfo.CephVersion.IsAtLeastNautilus() {
+		args = append(
+			args,
+			config.NewFlag("mds-standby-for-fscid", c.fsID),
+			config.NewFlag("mds-standby-replay", strconv.FormatBool(c.fs.Spec.MetadataServer.ActiveStandby)))
+	}
+
+	return args
+}
+
+func (c *Cluster) addMDSCacheMemLimit() []string {
+	var args []string
+
+	if !c.fs.Spec.MetadataServer.Resources.Limits.Memory().IsZero() && !c.clusterInfo.CephVersion.IsAtLeastMimic() {
```

Comment:
Mimic is already the min version. Did you mean to check for `IsAtLeastNautilus()` or can we remove this version check?

Reply:
Yes removing Mimic check.